gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -289,6 +346,36 @@ def main(args):
         ),
         action="store_true",
     )
+    parser.add_argument(
+        "--apply-models",
+        help="Apply plateau and coverage models to compute expected variant counts.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--partition-hint-when-counting-variants",
+        help=(
+            "Target number of partitions for aggregation when counting variants and"
+            " applying models."
+        ),
+        type=int,
+        default=2000,
+    )
+    parser.add_argument(
+        "--partition-hint-for-sum-aggregators",
```

Comment:
I felt that it's hard to make it more descriptive without making the name kind of long, but the variable is too wordy. Do you have any idea with the name?